### PR TITLE
Add functions to add and multiply ratios

### LINKED ISSUE #2850: Ratios: functions to add ratios and multiply ratios
## What is the Problem Being Solved?

In building dapp-token-economy/treasury, there were times when I wanted to add ratios together. For instance, to add a 2% buffer on top of a requirement of 150% collateralization without knowing the denominator of the collateralizationRatio ahead of time. (this ended up not getting into production, but you can imagine the use case). Also, @dtribble added a `composeRatio` function that he uses in swap. It'd be nice to add both addition and multiplication of ratios to the library.

### PR DESCRIPTION
**Pull request by Chris-Hibbert**
closes #2850

add `oneMinus()` as well.

### CODE REVIEW

File: packages/zoe/test/unitTests/contractSupport/test-ratio.js [hunk 1]
```diff
@@ -11,7 +11,9 @@ import {
   multiplyBy,
   divideBy,
   invertRatio,
+  oneMinus,
 } from '../../../src/contractSupport';
+import { multiplyRatios, addRatios } from '../../../src/contractSupport/ratio';
```
Comment (katelynsills) [inline packages/zoe/test/unitTests/contractSupport/test-ratio.js]: small thing, but can we import these from contractSupport as well?

File: packages/zoe/src/contractSupport/ratio.js [hunk 2]
```diff
@@ -141,3 +141,57 @@ export const invertRatio = ratio => {
...
+  assertIsRatio(left);
+  assert(
+    left.numerator.brand === left.denominator.brand &&
+      left.numerator.brand === right.numerator.brand &&
+      left.numerator.brand === right.denominator.brand,
+    X`all brands must match:  ${q(left)} ${q(right)}`,
+  );
+
+  return makeRatio(
+    multiply(left.numerator.value, right.denominator.value) +
```
Comment (katelynsills) [inline packages/zoe/src/contractSupport/ratio.js]: could we use the `natSafeMath.add` for consistency?

**Review by katelynsills — APPROVED** [review]
Looks good! I was comparing this to Dean's `composeRatio` function:
```js
const composeRatio = (x, y) =>
  makeRatioFromAmounts(multiplyBy(x.numerator, y), x.denominator);
```

It looks like it does intermediate rounding for no good reason, so I think this is much better.

File: packages/zoe/src/contractSupport/ratio.js (hunk 2 above)
Comment (Chris-Hibbert) [reply packages/zoe/src/contractSupport/ratio.js]: done

File: packages/zoe/test/unitTests/contractSupport/test-ratio.js (hunk 1 above)
Comment (Chris-Hibbert) [reply packages/zoe/test/unitTests/contractSupport/test-ratio.js]: done

**Pull request merged**